UPGRADE.md

```diff
@@ -33,16 +33,21 @@ With the release of Kuma 2.6.0, we've made some changes to the implementation of
 
 To ensure a smooth transition to Kuma 2.6.0, carefully review your existing configuration files and make necessary adjustments related to denied request responses and RBAC-related Envoy stats.
 
+### Make format SI valid for bandwidth in MeshFaultInjection policy
+
+Prior to this upgrade `mbps` and `gbps` were used for units for parameter `conf.responseBandwidth.percentage`.
+These are not valid units according to the [International System of Units](https://en.wikipedia.org/wiki/International_System_of_Units) they are respectively corrected to `Gbps` and `Mbps` if using
+these invalid units convert them into `kbps` prior to upgrade to avoid invalid format.
+
 ### Deprecation of postgres driverName=postgres (lib/pq)
 
 The postgres driver `postgres` (lib/pq) is deprecated and will be removed in the future.
 Please migrate to the new postgres driver `pgx` by setting `DriverName=pgx` configuration option or `KUMA_STORE_POSTGRES_DRIVER_NAME=pgx` env variable.
 
-### Make format SI valid for bandwidth in MeshFaultInjection policy
+### Deprecation of argument to transparent-proxy
 
-Prior to this upgrade `mbps` and `gbps` were used for units for parameter `conf.responseBandwidth.percentage`.
-These are not valid units according to the [International System of Units](https://en.wikipedia.org/wiki/International_System_of_Units) they are respectively corrected to `Gbps` and `Mbps` if using
-these invalid units convert them into `kbps` prior to upgrade to avoid invalid format.
+Parameter `--redirect-inbound-port-v6` is deprecated and will be removed in the future. The proxy will now redirect IPv6 traffic using the same port (15006) as ipv4 traffic, if you want to disable traffic redirection for IPv6 traffic, please use `--disable-ipv6`.
+Hence, the matching annotation `kuma.io/transparent-proxying-inbound-v6-port` is now deprecated and we've also added a new Kubernetes annotation `kuma.io/disable-ipv6` for disabling IPv6 traffic redirection on Kubernetes pods.
```

Comment:
Wdyt?

```suggestion
The `--redirect-inbound-port-v6` flag and the corresponding annotation `kuma.io/transparent-proxying-inbound-v6-port` are deprecated and will be removed in a future release of Kuma. These flags and annotations were used to configure the port used for redirecting IPv6 traffic to Kuma.

In the upcoming release, Kuma will redirect IPv6 traffic to the same port as IPv4 traffic (15006). This means that you no longer need to configure a separate port for IPv6 traffic.

If you want to disable traffic redirection for IPv6 traffic, you can use the `--disable-ipv6` flag.

We have also added a new Kubernetes annotation `kuma.io/disable-ipv6` for disabling IPv6 traffic redirection on Kubernetes pods. This annotation can be applied to any pod that you want to exclude from IPv6 traffic redirection.

We recommend that you update your configurations to use the new defaults for IPv6 traffic redirection. If you need to retain separate ports for IPv4 and IPv6 traffic, you can continue to use the deprecated flags and annotations until they are removed.
```

Reply:
I was aware the annotation was not intended to be set by the user, but by our `injector.go`, the user does not have a way to exclude certain Kubernetes pod.

Reply:
I'll update this section and reflect the latest code.